Rubberduck.CodeAnalysis/Inspections/Inspector.cs

```diff
@@ -53,7 +55,6 @@ private void UpdateInspectionSeverity(Configuration config)
                         if (inspection.Name == setting.Name)
                         {
                             inspection.Severity = setting.Severity;
-                            continue;
```

Comment:
Shouldn't this be a `break`?

Reply:
I don't even know where this is coming from ... let me investigate that bit of change

Reply:
seems to have been added on next with commit https://github.com/rubberduck-vba/Rubberduck/commit/459d322424721131e6b54b84422a339478dd7889 ... :cough: